README.md

```diff
@@ -1,2 +1,13 @@
 # ras-deploy
-RAS CD pipeline
+
+CI/CD pipeline for RAS services.
+
+![pipeline](https://i.imgur.com/HA4ENhA.png)
+
+## Deploying pipeline
+
+1. Copy `secrets.yml.example` to `secrets.yml`
+1. Assign values to all secret variables
+1. Login to concourse `fly -t lite login -c http://localhost:8080`
+1. Deploy the pipeline `./deploy-pipeline.sh`
+1. Go to [ras-deploy](http://localhost:8080/teams/main/pipelines/ras-deploy)
```

Comment:
I'd give the url rather than hide it behind the link so it can be cut/paste rather than just clicked through.

Also should probably list urls with the host variable as it won't necessarily be a local instance running on `8080`
e.g.
```
<concourse-host>/teams/main/pipelines/ras-deploy
```